src/PerformanceTests/Common/Scenarios/LoopRunner.cs

```diff
@@ -51,6 +51,8 @@ protected override Task Stop()
 
             const int MinimumBatchSeedDuration = 2500;
 
+            Log.InfoFormat("BatchHelper type: {0}", BatchHelper.Instance);
```

Comment:
did you intend to log the type?

Reply:
Yes, is seeding is slow you can verify in the log which batch sending style is used. I didn't put this in the batch helper itself as that is invoked in a loop which impacts the log size.

Reply:
Personally, I'd prefer if you make this explicit by passing the type instead of an object as the parameter to not rely on the actual `ToString` implementation

Reply:
Good suggestion, that better indicates intent.